.github/workflows/ci.yml

```
@@ -26,16 +30,235 @@ jobs:
 
-      - name: Build the tools
-        run: make all
+      - name: Setup vcpkg (Windows)
+        if: matrix.os == 'windows-latest'
+        run: |
+          git clone https://github.com/microsoft/vcpkg.git
+          cd vcpkg
+          .\bootstrap-vcpkg.bat
+          .\vcpkg.exe integrate install
+          cd ..
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)